docs/source/quickinstall.rst

```diff
@@ -95,6 +95,38 @@ This will take several minutes to run depending on your internet download speed.
 .. _options: https://docs.docker.com/engine/reference/commandline/run/
 .. _mounting volumes: https://docs.docker.com/storage/volumes/
 
+----------------------------------
+Installing from Source using Spack
+----------------------------------
+
+`Spack`_ is a package management tool designed to support multiple versions and
+configurations of software on a wide variety of platforms and environments.
+Please following Spack's `setup guide`_ to configure the Spack system.
+
+To install the latest OpenMC with GNU 9.3.0 without MPI:
+
+.. code-block:: sh
+
+    spack install openmc~mpi %gcc@9.3.0
+
```

Comment:
I think the first thing we should present to a user here in the quick install guide is the easiest, simplest way to install OpenMC (complete with Python API) if they don't care about MPI, compilers, etc.:
```
spack install py-openmc
```
After that, we can tell them how to customize if they need, e.g. by adding support for MPI, using a specific compiler, etc. I think we can leave that information in the full installation guide and for the quick install section, just put a link to the full guide.

Reply:
fixed